media/stylus/components/wiki/page-buttons.styl

```diff
@@ -14,6 +14,7 @@
         display: inline-block;
         position: relative;
         bidi-value(text-align, left, right);
+        color: $text-color;
```

Comment:
The problems with the styles originate with the .submenu class, let's associate this declaration with that class so it fixes the problems in other places it's used as well.

In mixin_submenu.styl could go: 
a) with the background-colour declaration in submenu-set
b) with the link color declaration in $submenu

a is better for maintainability
b is better for performance

If you don't have a preference I guess b is the right place for now.